test/extended/authorization/scopes.go

```diff
@@ -59,8 +59,9 @@ var _ = g.Describe("[sig-auth][Feature:OpenShiftAuthorization] scopes", func() {
 				t.Fatalf("unexpected error: %v", err)
 			}
 
+			privWhoamiOnlyToken, pubWhoamiOnlyToken := exutil.GenerateOAuthTokenPair()
 			whoamiOnlyToken := &oauthv1.OAuthAccessToken{
-				ObjectMeta:  metav1.ObjectMeta{Name: "whoami-token-plus-some-padding-here-to-make-the-limit-" + oc.Namespace()},
```

Comment:
where is "to make the limit" proporty gone? Do we lose anything? Or this just does not matter anymore because we always hash?

Reply:
the tokens have a requirement on at least 32 characters length, that's something we're keeping even though it may not make too much sense now that we're using hashed tokens